test/features/cross_device.feature

```diff
@@ -0,0 +1,19 @@
+@browser
+Feature: SDK Cross device steps
+
+  Scenario: Test cross device flow
+    Given I verify with passport
+    When I click on cross_device_button ()
+    Then page_title () should contain "Continue verification on your mobile"
+    When I open cross_device_link () in a new tab
+    Then page_title () should contain "Upload front of document"
+    When I switch to tab 1
+    Then page_title () should contain "Connected to your mobile"
+    When I switch to tab 2
+    And I upload my document and selfie
+    Then page_title () should contain "Uploads successful"
+    When I switch to tab 1
+    Then I wait for 1 second
```

Comment:
This might be error prone, you could add a new step to monster that waits until element contains text, something like:
```ruby
Then(/^I wait until #{element} contains "([^"]*)"$/) do |element, text|
  wait_until { element.text.include? text }
end
```

Reply:
The issue was the element did not exist yet. Will your fix cope with that?

Reply:
Given that #{element} uses transform function from fe-steps that retries while trying to find the element, it may cope. Not tested though

Reply:
it seems to work!

Reply:
I checked that locally several times and it works well. What exactly didn't work?
Weird if it works in 11th line but not in this one

Reply:
It worked for me locally too, but fails on travis.

On line 11 we show "connected to your mobile" before "upload front of document" is displayed on the other tab so that test has the order guaranteed.